### PR TITLE
(WIP) added the prevention of multiple-refunds for one day

### PR DESCRIPTION
**Pull request by Ayub-Khan**
Preventing Multiple Refunds for one day .

### CODE REVIEW

File: ecommerce/extensions/refund/models.py [hunk 1]
```diff
@@ -287,6 +291,14 @@ def deny(self):
 
         return result
 
+    def _can_issue_refund(self):
+        """ Disable the multiple refund for one day if already exists a payment event"""
+        refund_event = PaymentEventType.objects.get(name=PaymentEventTypeName.REFUNDED)
```
Comment (schenedx) [inline ecommerce/extensions/refund/models.py]: You don't need to make two database call here.
all you need is to do `event_type__name=PaymentEventTypeName.REFUNDED`

File: ecommerce/extensions/refund/models.py [hunk 2]
```diff
@@ -287,6 +291,14 @@ def deny(self):
...
+        return not (PaymentEvent.objects.filter(
+            order=self.order,
+            date_created__range=[datetime.timedelta(days=1), datetime.datetime.now()],
```
Comment (schenedx) [inline ecommerce/extensions/refund/models.py]: Please make this value configurable using a value in configuration model

File: ecommerce/extensions/refund/models.py [hunk 3]
```diff
@@ -233,6 +234,9 @@ def _revoke_lines(self):
             self.set_status(REFUND.REVOCATION_ERROR)
 
     def approve(self, revoke_fulfillment=True, notify_purchaser=True):
+        if not self._can_issue_refund():
+            logger.info('Refund [%d] have a existing refund Payment Event in last day.', self.id)
```
Comment (schenedx) [inline ecommerce/extensions/refund/models.py]: Update the comment with the time range from configuration model

**Review by schenedx — COMMENTED** [review]
The general direction looks good to me.

**Comment by Ayub-Khan** [comment]
moved to https://github.com/edx/ecommerce/pull/1490

**Pull request closed without merging**